FrontEnd/src/pages/homePage.js

```diff
@@ -4,54 +4,61 @@ import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 
 import 'materialize-css';
-import { Row, Col} from 'react-materialize';
+import { Row, Col } from 'react-materialize';
 
 import TitleBar from '../components/TitleBar';
 import SideBar from '../components/SideBar';
 import PageContent from '../components/PageContent';
 
 ///section css
 import mainCSS from '../index.scss';
-
+import CSS from './homePage.scss';
 
 ///section objects + components
 
-class homePage extends React.Component{
-
-    constructor(props) {
-        super(props);
-        this.state = {
-            pageTitle: this.props.pageTitle ? this.props.pageTitle : "PP",
-            testBox : ""
-        };
-    }
-
-    componentDidMount() {
-        document.title = this.state.pageTitle;  ///todo this will have to append the current navigationelement (try Redux?)
-    }
-
-    render(){
-        return(
-            <div className="homePage container page">
-                <Row className="full-height">
-                    <Col s={3} style={{ backgroundColor: "red" }} className="full-height"><SideBar/></Col>
-                    <Col s={9} className="full-height">
-                        <Row>
-                            <Col style={{ backgroundColor: "lightgreen" }} s={12}><TitleBar title="Project Patoune"/></Col>
-                        </Row>
-                        <Row>
-                            <Col style={{ backgroundColor: "blue" }} s={12}><PageContent /></Col>
-                        </Row>
-                    </Col>
-                </Row>
-            </div >
-
-        )
-    }
+class homePage extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			pageTitle: this.props.pageTitle ? this.props.pageTitle : 'PP',
+			testBox: '',
+		};
+	}
+
+	componentDidMount() {
+		document.title = this.state.pageTitle; ///todo this will have to append the current navigationelement (try Redux?)
+	}
+
+	render() {
+		return (
+			<div className="homePage page">
+				<Row className="full-height">
+					<div className="homePage__sidebar">
```

Comment:
maybe a silly question, but what's the point having a Row, if you put no Col in it?